src/widgets/header/header.tsx

```diff
@@ -42,7 +48,7 @@ export const Header: React.FunctionComponent = () => {
             sx={{
               mr: 2,
               display: { xs: 'none', md: 'flex' },
-              fontFamily: 'roboto',
+              fontFamily: 'Roboto',
               fontWeight: 700,
```

Comment:
Configure theme font and don't do it here

Reply:
fixed